frontend/src/models/conditions.ts

```diff
@@ -15,6 +15,33 @@ export const conditionTypes = [
   Enrg,
 ];
 
+// Options for condition indicator multiselect
+
+export const MultiSelectOptions = [
+  { value: 'ALL', label: 'ALL' },
+  { value: 'KPI', label: 'KPI' },
+  { value: 'DI', label: 'DI' },
+  { value: 'IRI', label: 'IRI' },
+  { value: 'Mu', label: 'Mu' },
+  { value: 'E_norm', label: 'E_norm' },
+];
+
+// MultiMode manager
+
+export interface MultiMode {
+  count?: number;
+  mode?: any;
```

Comment:
```suggestion
  mode: string[];
```
A suggestion, you could have the count recomputed with `mode.length()`